tests/html_glimmer/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -0,0 +1,493 @@
+<user-greeting @name="Ricardo" @greeting="Olá"></user-greeting>
+{{@greeting}}
+,
+{{@name}}
+!
+<button onclick={{action next}}>
+  Next
+</button>
+`;
+
+exports[`concat-statement.hbs 1`] = `
+<div class="hello {{if goodbye true}}">
+  Hello
+</div>
+
+<div class="hello {{if goodbye true}} {{if goodbye false}} {{if goodbye true}} {{if goodbye false}} {{if goodbye true}}">
+  Hello
+</div>
+
+<a href="/{{url}}/{{url}}"></a>
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+<div class="hello{{if goodbye true}}">
+  Hello
+</div>
+<div
+  class="hello
+    {{if goodbye true}}
+    {{if goodbye false}}
+    {{if goodbye true}}
+    {{if goodbye false}}
+    {{if goodbye true}}"
+>
+  Hello
+</div>
+<a href="/{{url}}/{{url}}"></a>
+`;
+
+exports[`element-modifier-statement.hbs 1`] = `
+<div {{hello param hash=key}} {{goodbye param}}>
+  Hello
+</div>
+
+<div {{hello param param param param param param param param param param param param}}>
+  Hello
+</div>
+
+<div {{hello hashPair=value hashPair=value hashPair=value hashPair=value hashPair=value}}>
+  Hello
+</div>
+
+<div {{hello param param param param hashPair=value hashPair=value hashPair=value hashPair=value hashPair=value}}>
+  Hello
+</div>
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+<div {{hello param hash=key}} {{goodbye param}}>
+  Hello
+</div>
+<div
+  {{hello
+    param
+    param
+    param
+    param
+    param
+    param
+    param
+    param
+    param
+    param
+    param
+    param
+  }}
+>
+  Hello
+</div>
+<div
+  {{hello
+    hashPair=value
+    hashPair=value
+    hashPair=value
+    hashPair=value
+    hashPair=value
+  }}
+>
+  Hello
+</div>
+<div
+  {{hello
+    param
+    param
+    param
+    param
+    hashPair=value
+    hashPair=value
+    hashPair=value
+    hashPair=value
+    hashPair=value
+  }}
+>
+  Hello
+</div>
+`;
+
+exports[`element-node.hbs 1`] = `
+<div class="attribute" {{modifier}} {{! comment}}>
+  Hello
+</div>
+
+<div>
+  Hello
+</div>
+
+<div>
+  hi
+</div>
+
+<div>
+  A long enough string to trigger a line break that would prevent wrapping.
+</div>
+
+<div>
+  A long enough string to trigger a line break that would prevent wrapping more.
+</div>
+
+<div>
+  A long enough string to trigger a line break that would prevent wrapping more and more.
+</div>
+
+<div>
+  {{#block}}
+    {{hello}}
+  {{/block}}
+</div>
+
+<div>
+  {{hello}}
+</div>
+
+<div></div>
+<img />
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+<div class="attribute" {{modifier}} {{! comment}}>
+  Hello
+</div>
+<div>
+  Hello
+</div>
+<div>
+  hi
+</div>
+<div>
+  A long enough string to trigger a line break that would prevent wrapping.
+</div>
+<div>
+  A long enough string to trigger a line break that would prevent wrapping more.
+</div>
+<div>
+  A long enough string to trigger a line break that would prevent wrapping more and more.
```

Comment:
You're not wrapping here?

Reply:
This is one the tricky ones to decide. You wouldn't break this string up to multiple lines:
```js
const hello = 'A string that is over 80 chars long';
```
I could take text and break on space, but I think that's against what I would expect from prettier.

Reply:
I would recommend looking at what prettier outputs for jsx and try to be as close as possible. This has been battle tested over time and it hasn’t been a big source of issues.